pageserver/src/basebackup.rs

```diff
@@ -133,6 +134,84 @@ where
     ctx: &'a RequestContext,
 }
 
+/// A sink that accepts SLRU blocks ordered by key and forwards
```

Comment:
FYI: "sink" has specific meaning inside the rust async ecosystem: https://docs.rs/futures-sink/latest/futures_sink/trait.Sink.html

Reply:
Yeah, I stumbled upon `Sink` previously. I was hoping there's enough context for it to be clear I'm referring to the generic sink.